extension_versions.csv

```diff
@@ -81,10 +81,22 @@ signatories,,master,https://raw.githubusercontent.com/open-contracting-extension
 transaction_milestones,2017-05-09,v1.1,https://raw.githubusercontent.com/open-contracting-extensions/ocds_transactions_relatedMilestone_extension/v1.1/,https://api.github.com/repos/open-contracting-extensions/ocds_transactions_relatedMilestone_extension/zipball/v1.1
 unstructuredChanges,,master,https://raw.githubusercontent.com/open-contracting-extensions/ocds_unstructuredChanges_extension/master/,https://github.com/open-contracting-extensions/ocds_unstructuredChanges_extension/archive/master.zip
+bids,2018-01-30,v1.1.3,https://raw.githubusercontent.com/open-contracting-extensions/ocds_bid_extension/v1.1.3/,https://api.github.com/repos/open-contracting-extensions/ocds_bid_extension/zipball/v1.1.3
+bids,2017-08-07,v1.1.1,https://raw.githubusercontent.com/open-contracting-extensions/ocds_bid_extension/v1.1.1/,https://api.github.com/repos/open-contracting-extensions/ocds_bid_extension/zipball/v1.1.1
+bids,2017-05-09,v1.1,https://raw.githubusercontent.com/open-contracting-extensions/ocds_bid_extension/v1.1/,https://api.github.com/repos/open-contracting-extensions/ocds_bid_extension/zipball/v1.1
+enquiries,2018-02-01,v1.1.3,https://raw.githubusercontent.com/open-contracting-extensions/ocds_enquiry_extension/v1.1.3/,https://api.github.com/repos/open-contracting-extensions/ocds_enquiry_extension/zipball/v1.1.3
+location,2018-02-01,v1.1.3,https://raw.githubusercontent.com/open-contracting-extensions/ocds_location_extension/v1.1.3/,https://api.github.com/repos/open-contracting-extensions/ocds_location_extension/zipball/v1.1.3
+location,2017-08-07,v1.1.1,https://raw.githubusercontent.com/open-contracting-extensions/ocds_location_extension/v1.1.1/,https://api.github.com/repos/open-contracting-extensions/ocds_location_extension/zipball/v1.1.1
+lots,2018-01-30,v1.1.3,https://raw.githubusercontent.com/open-contracting-extensions/ocds_lots_extension/v1.1.3/,https://api.github.com/repos/open-contracting-extensions/ocds_lots_extension/zipball/v1.1.3
+lots,2017-08-07,v1.1.1,https://raw.githubusercontent.com/open-contracting-extensions/ocds_lots_extension/v1.1.1/,https://api.github.com/repos/open-contracting-extensions/ocds_lots_extension/zipball/v1.1.1
+lots,2017-05-09,v1.1,https://raw.githubusercontent.com/open-contracting-extensions/ocds_lots_extension/v1.1/,https://api.github.com/repos/open-contracting-extensions/ocds_lots_extension/zipball/v1.1
+process_title,2017-08-07,v1.1.1,https://raw.githubusercontent.com/open-contracting-extensions/ocds_process_title_extension/v1.1.1/,https://api.github.com/repos/open-contracting-extensions/ocds_process_title_extension/zipball/v1.1.1
+process_title,2017-05-09,v1.1,https://raw.githubusercontent.com/open-contracting-extensions/ocds_process_title_extension/v1.1/,https://api.github.com/repos/open-contracting-extensions/ocds_process_title_extension/zipball/v1.1
```

Comment:
How did these end up here? I removed these in c54a52c6243e52c8a437659b5c8b840aa09dbd2d and bfe08e9713a9882212fc394cdcbf2e23a088bcc1

Reply:
I ran `./manage.py refresh`

Reply:
Ah, I guess I'll add some exceptions to it.

Reply:
For now, please delete them from the PR.

Reply:
Donw